jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+---
+layout: classic-docs
+title: "Bundling Config"
+short-title: "Bundling Config"
+description: "Bundling configuration with CircleCI 2.1"
+categories: [configuration]
+order: 1
+---
+
+
+This document describes how to version your [.circleci/config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file and get started with bundling config.
+
+* TOC
+{:toc}
+
+## Getting Started with Config Bundling
```

Comment:
This feels heavy -- we don't talk about how to start projects for other config features. Is the plan that this is the only place we'll talk about these new config elements? If they aren't visible on config reference they end up on an island that seems odd.